stdlib/LinearAlgebra/src/triangular.jl

```diff
@@ -295,6 +300,10 @@ end
     return A
 end
 
+Base._reverse(A::LowerTriangular, dims) = LowerTriangular(reverse((Matrix(A)); dims))
```

Comment:
```suggestion
Base._reverse(A::LowerTriangular, dims) = reverse(Matrix(A); dims)
```
The output is not lower triangular